server/src/main/java/org/spine3/server/event/EventStorage.java

```diff
@@ -57,11 +66,31 @@ public Event apply(@Nullable EventEntity input) {
                 }
             };
 
+    @Override
+    protected org.spine3.server.storage.EventStorage createStorage(StorageFactory factory) {
```

Comment:
We should for sure but we cannot avoid usage of FQN if the qualified class has the same short name as the one that uses it.